Automattic/jetpack
# Social | Migrate remaining settings to script data

This is the last PR for our initial state migration 🎉 

## Proposed changes:

* Remove the `siteData` related reducer and selectors
* Flip the initial state for the social store from old data to the new script data
* Clean things up a bit 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Disconnect Jetpack from My Jetpack, if already connected
* Goto Social admin page
* Click Get Started and complete the connection
* Confirm that the connection is successful
* From the Jetpack Settings page, Social admin page and the editor, cConfirm that the connections are loaded fine on the initial page load
* Confirm that share status loads fine
